packages/blocks/src/core-embed/index.js

```diff
@@ -22,20 +25,17 @@ const getPhotoHtml = ( photo ) => {
 const CoreEmbed = ( { attributes } ) => {
 	const { url, className, caption, type, providerNameSlug } = attributes;
 
-	const { preview } = useSelect(
-		( select ) => {
-			const embedPreview = select( coreStore ).getEmbedPreview( url );
-			const badEmbedProvider =
-				embedPreview?.html === false &&
-				embedPreview?.type === undefined;
-			const validPreview = !! embedPreview && ! badEmbedProvider;
+	const [ preview, setPreview ] = useState( {} );
 
-			return {
-				preview: validPreview ? embedPreview : {},
-			};
-		},
-		[ url ]
-	);
+	useEffect( () => {
+		fetchEmbedContent( { url } ).then( ( { data } ) => {
```

Comment:
On WP, there is a piece of code that removes a wrapper div from the response. Ex `<div class="embed-youtube">`.
This is necessary to prevent some styling issues.
On this PR, if you embed a YouTube video and take a look at it on the Project Renderer, it'll be cropped.
When I first implemented the embed block support, I added this piece of code inside the [embedRequestInterceptor](https://github.com/Automattic/crowdsignal-ui/blob/main/packages/rest-api/src/embed/index.js#L14), since we were using the WP components, both requests, for the Editor and for the Project Renderer were intercepted and handled properly.
Now that we're using `fetchEmbedContent` directly, we'll need to adapt the code to remove the wrapper div.